scripts/deploy_gh_pages.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+git clone -b gh-pages https://${GH_TOKEN}@github.com/stackstorm-japan/st2docs.jp.git gh-pages
```

Comment:
We need the description what kind of value is set in the variable `GH_TOKEN` and an error handler of this processing.